doc/rados/operations/pools.rst

```diff
@@ -188,7 +188,7 @@ consider removing them when you no longer need your pool::
 
 	ceph osd pool get {pool-name} crush_ruleset
 
-If the ruleset was "123", for example, you can check the other pools like so::
+If the ruleset ID was "123", for example, you can check the other pools like so::
```

Comment:
Got it, only found in release-notes by now, get those using: `grep -Ri 'ruleset id' doc/`